### DISCUSSION TITLE
Bootstrap JS Import

### DISCUSSION THREAD

**Discussion by LoganC034** [sveltejs/kit#7680] [General]
Given the issues w/ sveltekit & sveltestrap compatibility, has anyone come up with a production worthy solution to import bootstrap.js so that things like dropdowns will work?  

https://github.com/bestguy/sveltestrap/issues/517

**Comment by Hejtmus** [comment]
Hey, 
I had same issue (caused by PopperJS) and found [fork of sveltestrap](https://github.com/ClapClap-app/sveltestrap) which makes it usable with SvelteKit. It has also package on GH packages, you can install it like this:

```
npm i github:ClapClap-app/sveltestrap -D
```
